### PR TITLE
Feature/17/add footer

### LINKED ISSUE #17: Add Footer

### PR DESCRIPTION
**Pull request by aidantrabs**

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
+import { FaXTwitter, FaLinkedin } from "react-icons/fa6";
+import { FaInstagram } from "react-icons/fa";
+import { BsChat } from "react-icons/bs";
+
+import { footerIcon } from "@assets";
+
+const Footer = () => {
+    return (
+        <div>
+            <hr className="border-t-1 border-gray-200 mt-12 mb-12" />
```
Comment (AmirAgassi) [inline src/components/Footer.tsx]: LGTM, but maybe copy the color from the other hr? To match the figma

```suggestion
            <hr className="border-t-1 border-[#3D3D3D] mt-12 mb-12" />
```

**Review by AmirAgassi — DISMISSED** [review]

**Review by AmirAgassi — DISMISSED** [review]
LGTM!

**Review by juancwu — DISMISSED** [review]
lgtm :+1:

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                            Canada
+                        </p>
+
+                        <a 
+                            href="https://maps.app.goo.gl/tHhBEYXZiyr5hpgq6"
+                            target="_blank"
+                            rel="noreferrer"
+                            className="text-sm lg:text-md text-white hover:underline"
+                        >
+                            69 Breithaupt St, <br />
```
Comment (juancwu) [inline src/components/Footer.tsx]: Use 283 Duke St West

**Review by juancwu — DISMISSED** [review]

**Pull request merged**